google/ggrc-core
# GGRC-6208 Remove not used flask-jasmine dependency

# Issue description

FE uses own test runner for Jasmine and there is no usages of Flask-Jasmine anywhere.

# Steps to test the changes

Test steps 1:
- run `build_assets`
- run `launch_gae_ggrc`

Test steps 2:
- run `build_assets`
- run `launch_ggrc`

Expected result: no regression during launch run

# Solution description

Remove Flask-Jasmine from BE dependencies.

# Sanity checklist




# PR Review checklist
